controllers/issue.js

```diff
@@ -0,0 +1,40 @@
+const FacadeIssueController = require('../facade/issue.js');
```

Comment:
It's a bit odd to call facade module with controller. Should be just FacadeIssue